lib/index.js

```diff
@@ -255,6 +258,10 @@ Server.prototype = {
       });
       yield this.apiml.registerMainServerInstance();
     }
+    bootstrapLogger.log(bootstrapLogger.INFO,
```

Comment:
This is hit asynchronously to the events above, so it cannot be correct that the server is actually ready at this time.